libr/core/cmd_debug.c

```diff
@@ -51,7 +51,7 @@ static const char *help_msg_db[] = {
 	"dbj", "", "List breakpoints in JSON format",
 	// "dbi", " 0x848 ecx=3", "stop execution when condition matches",
 	"dbc", " <addr> <cmd>", "Run command when breakpoint is hit",
-	"dbC", " <addr> <cmd>", "Set breakpoint condition on command",
+	"dbC", " <addr> <cmd>", "Run command when breakpoint is hit, but continue until condition on command returns zero",
```

Comment:
too long, try to fit in the same length as other messages, should fit in 80 columns.